test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -951,6 +951,7 @@ def test_mom_down(self):
             overwrite=True)
 
         a = {'Resource_List.select': '3:ncpus=1',
+             'Resource_List.walltime': 300,
              'Resource_List.place': 'scatter'}
         j = Job(TEST_USER)
         j.set_attributes(a)
```

Comment:
Don't you need to increase job's sleep time as well to 300? else even if u have job with 300s walltime still it will finish in 100s as by default job runs `/bin/sleep 100`

Reply:
I agree Hiren, thanks for catching it.